apiml-sample-extension/README.md

```diff
@@ -0,0 +1,10 @@
+# API ML sample extension
+
+This is an API ML sample extension. It only contains a simple controller for testing.
+The extension is added to the API Gateway class path.
```

Comment:
It would be good to explain a bit more what does it mean. E.g. that it adds the controller in the context of the Gateway Service without starting new service.